cmd/up/exec.go

```diff
@@ -220,9 +220,10 @@ func (eg *envsGetter) getEnvs(ctx context.Context) ([]string, error) {
 		return nil, err
 	}
 
-	imageEnvs, err := eg.imageEnvsGetter.getEnvsFromImage(apps.GetDevContainer(app.PodSpec(), "").Image)
+	svcImage := apps.GetDevContainer(app.PodSpec(), "").Image
+	imageEnvs, err := eg.imageEnvsGetter.getEnvsFromImage(svcImage)
 	if err != nil {
-		return nil, err
+		oktetoLog.Warning("Could not to retrieve environment variables from the image '%s'", svcImage)
```

Comment:
Can we also log the original error in debug level?

Reply:
06ecfc7d7a4f4651111d0aee999fa24a8ef21c3a